src/project-tests/markdown-previewer-tests.js

```diff
@@ -8,15 +8,43 @@ export default function createMarkdownPreviewerTests() {
     // Save the values of the editor and preview.
     const editor = document.getElementById('editor');
     const preview = document.getElementById('preview');
+    
+    /**
+     * Using class PreviewDocument to be a proxy for grabing the document
+     * information. In case this is an iframe, use the contentDocument of
+     * the iframe.
+     */
+    class PreviewDocument {
+      static get document() {
+        if (preview && preview.contentDocument) {
+          return preview.contentDocument.querySelector('div#root div');
```

Comment:
Will this be a customary arrangement for campers choosing to use an iframe? Will we need to introduce a test for the existence of the `div#root div` element?